assets/styles/layout/_options.scss

```diff
@@ -123,6 +154,25 @@
     text-transform: uppercase;
   }
 
+  .hide-on-mobile {
+    @include media-breakpoint-down(lg) {
+      display: none;
+    }
+  }
+
+  &__filter button {
+    font-size: 0;
+
+    i {
+      font-size: .85rem;
+    }
+
+    span {
+      font-size: .85rem;
+      margin-left: 0.5rem;
+    }
+  }
```

Comment:
There already is a selector `.options __filter button` in line 88, please merge it with that

Reply:
Done 👍🏻